src/daft-sql/src/functions.rs

```diff
@@ -46,9 +49,29 @@ pub(crate) static SQL_FUNCTIONS: LazyLock<SQLFunctions> = LazyLock::new(|| {
     functions.register::<SQLModuleConfig>();
     functions.register::<SQLModuleWindow>();
     functions.add_fn("coalesce", SQLCoalesce {});
+    for (name, function) in FUNCTION_REGISTRY.entries() {
```

Comment:
once all other modules are registered in `FUNCTION_REGISTRY`, we can combine these in to a single struct. (SQL_FUNCTIONS & FUNCTION_REGISTRY)